src/Features/Core/Portable/Completion/CompletionService.cs

```diff
@@ -289,25 +289,18 @@ public virtual TextSpan GetDefaultCompletionListSpan(SourceText text, int caretP
             // i.e. when everything else is equal, then if item1 is a better case-sensitive match but item2 has higher 
             // MatchPriority, we consider them equally good match, so the controller will later have a chance to
             // decide which is the best one to select.
-            var filterTextContainsNoUpperLetters = true;
+            var filterTextIsLowerCaseOnly = true;
             for (var i = 0; i < filterText.Length; ++i)
             {
                 if (char.IsUpper(filterText[i]))
                 {
-                    filterTextContainsNoUpperLetters = false;
+                    filterTextIsLowerCaseOnly = false;
```

Comment:
this name doesn't seem accurate.  "lower case only" would mean `.All(char.IsLower)` but char.isLower and char.isUpper at not opposites.  So this code really is saying that we have no upper characters (which doesn't mean we only have lwoer characters).

Reply:
Will fix in a follow up PR.